x-pack/plugins/cross_cluster_replication/server/types.ts

```diff
@@ -21,10 +21,24 @@ export interface Dependencies {
 }
 
 export interface RouteDependencies {
-  router: IRouter;
+  router: CcrPluginRouter;
   license: License;
   lib: {
     isEsError: typeof isEsError;
     formatEsError: typeof formatEsError;
   };
 }
+
+/**
+ * @internal
+ */
+export interface CcrRequestHandlerContext extends RequestHandlerContext {
+  crossClusterReplication: {
+    client: ILegacyScopedClusterClient;
+  };
+}
+
+/**
+ * @internal
+ */
+export type CcrPluginRouter = IRouter<CcrRequestHandlerContext>;
```

Comment:
nit; is this being used outside of this file?

Reply:
nope. I will remove `export`